package.json

```
@@ -40,6 +40,9 @@
 		"typescript": "^5.0.2",
 		"xo": "^0.53.1"
 	},
+	"peerDependencies": {
+		"typescript": "^4.7.0 || ^5.0.0"
```

```suggestion
		"typescript": ">=4.7.0"
```

We should allow any version from 4.7.0 and above. I realize a TS v6 is far off, but still.